src/ansys/fluent/core/launcher/standalone_launcher.py

```diff
@@ -236,14 +236,7 @@ def __init__(
         )
 
         if is_windows():
-            if (
-                pyfluent.config.launch_fluent_stdout
-                or pyfluent.config.launch_fluent_stderr
-            ):
-                self._launch_cmd = self._launch_string
-            else:
-                # Using 'start.exe' is better; otherwise Fluent is more susceptible to bad termination attempts.
-                self._launch_cmd = 'start "" ' + self._launch_string
+            self._launch_cmd = self._launch_string
```

Comment:
@mkundu1, this part was the one causing issue. Adding "start" to run it via start.exe needed the shell to be True, and with shell=True there was no option to make the window hidden. This fix works but I wanted to check the reason for introducing "start" in the first place. Can we work with this fix or is there a way to use start and shell=False simultaneously?

cc. @seanpearsonuk